installer/lib/phx_new/web.ex

```diff
@@ -28,6 +28,10 @@ defmodule Mix.Tasks.Phx.New.Web do
   use Phx.New.Generator
   alias Phx.New.{Project}
 
+  @version Mix.Project.config[:version]
+  @shortdoc "Creates a new bare Phoenix v#{@version} application without database integration inside an umbrella application"
```

Comment:
The shortdoc needs to be shorter. :) Also, no need for that many new lines afterwards.